gateway/examples/config.toml

```diff
@@ -2,6 +2,12 @@
 # Oxide API: example configuration file
 #
 
+# Maximum number of host phase2 trampoline images we're willing to cache. Note
+# that this value is specified in terms of _number of images_, not bytes, and
+# our cache is in-memory. We expect this value to be small in production,
+# potentially even 1 (i.e., only keep the most-recently-uploaded image).
+host_phase2_recovery_image_cache_max_images = 1
```

Comment:
Trampoline phase 2. This is probably a copy/paste from when I was calling it "recovery image" instead of "trampoline image"; I should make a pass through MGS and replace all of those at some poitn.